### DISCUSSION TITLE
Future Plans for Ultimate Vocal Remover

### DISCUSSION THREAD

**Discussion by FSharpCSharp** [Anjok07/ultimatevocalremovergui#1142] [General]
Dear @Anjok07,

I hope this message finds you well. I am writing to inquire about the development plan for the "Ultimate Vocal Removers" project. It has been four months since the last commit, and I am curious to know if the project is still actively maintained.

Will there be new releases with new models or features in the near future? I am a big fan of this project as it significantly simplifies my daily routine and I benefit greatly from it. However, the fact that the last commit was quite some time ago makes me a bit sad.

I want to emphasize how brilliant I find this project. It has positively influenced my work and life to such an extent that I can hardly imagine doing without it. It would be a real pity if such a wonderful project were to end. Therefore, I kindly ask you to share some information about the project's status, even if it's just a few words. It would be incredibly reassuring to know that there is still activity and that the project has not been discontinued.

I thank you for your time and for the amazing work you have put into this project. Your efforts have made a big difference and I look forward to seeing what the future holds for "Ultimate Vocal Removers".

**Comment by FSharpCSharp** [comment]
Dear @Anjok07 and @aufr33,

I trust this message finds you both in good health and spirits. I am writing to you again regarding the "Ultimate Vocal Removers" project. It has been a couple of weeks since my last inquiry, and I am still very interested in learning about any updates or plans for the project.

I understand that maintaining an open-source project requires significant time and effort, and I appreciate all the work that has been put into it so far. This project has had a profound impact on my daily routine, and I am sure many others feel the same way. The excellent quality of the product and its unique features have made it an indispensable tool for me.

However, the recent silence around the project has left me a bit concerned. I am aware that there aren't many open-source projects in this field, and it would be a great loss if this project were to cease. Therefore, I kindly request any information, even if brief, about the current status and future plans for "Ultimate Vocal Removers".

Knowing that the project is still active and not discontinued would provide immense reassurance to myself and likely many others who rely on this tool. I am hopeful for a positive update and look forward to the continued success of "Ultimate Vocal Removers".

Thank you once again for your time and for the incredible work you have done on this project. Your efforts have truly made a difference.

Best regards,
FSharpCSharp

**Comment by aufr33** [comment]
Thank you for your feedback. The project continues to evolve. We will be adding some new models soon.

**Comment by Anjok07** [comment]
@FSharpCSharp Thank you for your kind words! Things have been quite busy on my end, so I haven't been able to spend as much time on UVR as I'd like. However, as @aufr33 said, the project is evolving and still alive and well. New models are in development, and a new update to the GUI with new features (including a demudder) and bug fixes will be released sometime in the next few months.

**Comment by FSharpCSharp** [comment]
Dear @Anjok07 and @aufr33,

Firstly, I would like to express my gratitude for your responses. It is reassuring to know that the development of this open-source project continues.

However, I have some concerns that I need to address regarding the future of the project. Over the weekend, I discovered a project named UVR-Cloud, which seems to be based on this open-source project. It appears to be offering the same services as the original UVR program through subscriptions. This worries me as it seems the project is becoming more commercialized. I understand that operating the project and the costs of AI training can be significant. However, I am uncomfortable with the fact that the services are now exclusively provided through the cloud.

Therefore, my urgent request is that the AI models continue to be downloadable and directly provided in the program. I am willing to pay for the AI models if the quality is higher, as with the VIP models. It is very important to me that the services are not only offered via the cloud, where one does not know what happens to the uploaded data. Hence, my question is whether the local project will continue to be developed, and whether the AI models currently available in the cloud (perhaps with a delay) can be downloaded and used in the local program.

Thank you once again for your time and for the incredible work you have done on this project. Your efforts have truly made a difference.

Best regards,
FSharpCSharp

**Reply by FSharpCSharp** [reply]
Dear @Anjok07 and @aufr33,

I hope this message finds you well. I am writing to follow up on my previous inquiry regarding the future of our open-source project. I understand that you are both very busy, but I am still concerned about the project's direction, especially since there hasn't been much activity on the code recently.

As I mentioned in my last message, I discovered a project named UVR-Cloud, which seems to be based on our open-source project and offers similar services through subscriptions. This development has raised some concerns for me about the project becoming more commercialized and the services being exclusively provided through the cloud.

My urgent request is for the AI models to continue being downloadable and directly provided in the program. I am willing to pay for the AI models if the quality is higher, as with the VIP models. It is crucial to me that the services are not only offered via the cloud, where the data handling remains unclear.

Therefore, I would appreciate it if you could provide some clarity on whether the local project will continue to be developed, and whether the AI models currently available in the cloud (perhaps with a delay) can be downloaded and used in the local program.

Thank you once again for your time and the incredible work you have done on this project. Your efforts have truly made a difference.

Best regards,
FSharpCSharp

**Comment by aufr33** [comment]
> I discovered a project named UVR-Cloud

Could you please clarify what a UVR-Cloud is?

**Reply by FSharpCSharp** [reply]
@aufr33 
I stumbled upon the UVR-Cloud project, which is an online version of the Ultimate Vocal Remover GUI application for music and vocal separation. It's available at https://www.patreon.com/uvronline. The project aims to provide a platform for music and vocal separation with enhanced features for its subscribers.

Once you subscribe, you gain access to a post containing a secret Premium link. The Premium version offers a range of features that surpass those of the free version. These include:

- Improved quality processing with the latest models.
- The ability to separate into lead/backing vocals, bass, drums, guitar, or saxophone.
- Models for de-noise, de-echo, and de-reverb.
- Larger limits (depending on the subscription level), allowing for more audio minutes and thus more songs to be processed.
- The option to download in WAV / FLAC formats.

Moreover, they already provide additional models for supporters or subscribers that seem to be absent in the offline version of the "Ultimate Vocal Removers".

To be honest, I'm not entirely sure how this project relates to the "Ultimate Vocal Remover", and whether an offline version could still be provided. I have reservations about cloud-based services and would prefer to have the complete open-source product installed on my hard drive. I believe it's always important to ensure the software you use respects your privacy and security. If an offline version of the Ultimate Vocal Remover is available and meets your needs, it would indeed be a good choice. I hope this helps clarify things a bit. If you have any other questions, feel free to ask.

**Comment by aufr33** [comment]
@FSharpCSharp This project is under my supervision, new models usually appear there first. However, you have nothing to worry about, you will see them in the UVR GUI after a while. My De-crowd model has already been added. The delay is due to some sites using our models without crediting us. Besides, UVR is my main and only job. I am not employed anywhere and dedicate almost all my free time to this project. 

Thanks for your understanding.

**Reply by FSharpCSharp** [reply]
Dear @aufr33,

I want to express my deepest gratitude for your detailed explanation and the transparency you've provided. It's truly commendable that you're dedicating your time and energy to this project as your primary occupation. Your commitment to this project is not only admirable but also inspiring.

The fact that you're not employed elsewhere and are devoting almost all your free time to this project speaks volumes about your passion and dedication. It's clear that this isn't just a job for you, but a mission that you're deeply committed to. This level of dedication is rare and highly commendable.

I also appreciate your efforts to ensure that the models and features can be used offline and without a cloud connection. This consideration for the users' needs and preferences is a testament to your user-centric approach, which is greatly appreciated.

I've had the opportunity to review the De-Crowd model and I must say, I found it to be extremely useful. The model's ability to de-crowd audio is impressive and it has been a valuable tool in my work. I'm curious to know if there will be an improved or second version of this model? As we all know, machine learning models can sometimes be a hit or miss, but with your expertise and dedication, I'm confident that any future versions will only be better.

Lastly, I want to reiterate my willingness to support this project financially. I believe in the value of your work and I'm more than happy to contribute towards future models or features, as long as I can download and use them offline or without a cloud connection. I look forward to seeing what you come up with next.

Once again, thank you for your outstanding work on this project. Your dedication and commitment are truly inspiring.

Best,
FSharpCSharp

**Reply by aufr33** [reply]
> I'm curious to know if there will be an improved or second version of this model?

Yes, it's a possibility in the future, but we're going to release soon:

1. Piano model
2. Synth model
3. Vocal remover model

**Reply by FSharpCSharp** [reply]
Thank you for the update on the upcoming models. It's truly exciting to hear about the imminent release of the Piano model, Synth model, and Vocal remover model. These models, each with their unique capabilities, are a testament to the rapid advancements in machine learning and artificial intelligence. The pace at which these models are being developed and released is nothing short of impressive.

I'm particularly interested in knowing if there are any plans to release new models specifically for Speech Enhancement or Music Enhancement. The potential of these models to significantly improve the quality of audio recordings is immense. The idea of "Super Resolution" models that could restore higher frequencies in audio recordings is fascinating. Imagine being able to enhance an 8000 - 11025 Hz speech or music recording, or upgrading a 22050 Hz music recording to 44100 Hz. The possibilities are endless.

What's even more remarkable is that these complex models are being made available for free. This is a significant feat considering the substantial resources, both in terms of computational power and human expertise, required to train them. The cost of training such models can be quite high, yet here they are, being offered at no cost. This is a clear demonstration of the commitment to making advanced machine learning models accessible to everyone.

Your respect and appreciation for this effort are greatly valued. It's heartening to see such enthusiasm and understanding for the work that goes into developing these models. Your words of praise serve as a motivation for those working tirelessly behind the scenes to bring these models to life.

Looking forward to hearing more about the future developments in this space. The anticipation for what's to come is palpable. With the rapid pace of advancements, who knows what amazing models we might see in the near future. Thanks again for your kind words and interest in these developments!

**Reply by Belissimo-T** [reply]
@FSharpCSharp Your writing sounds and looks a lot like from a LLM and I found it quite irritating to read bc a lot of it is just unnecessary words. I'd prefer a shorter and more concise text.

**Comment by ybhka2020** [comment]
Hello: @aufr33, please tell me the SEGMENT SIZE and OVERLAP of the BS1296 model. What are the specific settings? I mainly extract the accompaniment.

**Comment by biglexj** [comment]
Alguna novedad 😇
